README.md

```
@@ -423,6 +423,20 @@ npm run materialize:emilia-map -- --data-root C:\Users\dacan\GeoLens\data\emilia
 npm run verify:emilia-map -- --data-root C:\Users\dacan\GeoLens\data\emilia-romagna-2023
 ```
 
+## External evidence intake
+
+GeoLens includes one local intake command for future ARPAE and Amsterdam/Waternet deliveries. It reads originals from a data directory outside Git, computes their byte counts and SHA-256 identities, validates the appropriate scientific contract and writes a new package receipt. It does not copy the originals and refuses to overwrite an existing receipt.
+
+```powershell
```

markdownlint-cli2 (0.23.2)

[warning] Code fence style
Expected: tilde; Actual: backtick

(MD048, code-fence-style)